frappe/public/js/frappe/form/controls/duration.js

```diff
@@ -0,0 +1,140 @@
+frappe.ui.form.ControlDuration = frappe.ui.form.ControlData.extend({
+	make_input: function() {
+		this._super();
+		this.make_picker();
+	},
+
+	make_picker: function() {
+		this.inputs = [];
+		this.set_duration_options();
+		this.$picker = $(
+			`<div class="duration-picker">
+				<div class="picker-row row"></div>
+			</div>`
+		);
+		this.$wrapper.append(this.$picker);
+		this.build_numeric_input("days", !this.duration_options.showDays);
+		this.build_numeric_input("hours", false);
+		this.build_numeric_input("minutes", false);
+		this.build_numeric_input("seconds", !this.duration_options.showSeconds);
```

Comment:
```suggestion
		this.build_numeric_input("days", !this.duration_options.show_days);
		this.build_numeric_input("hours", false);
		this.build_numeric_input("minutes", false);
		this.build_numeric_input("seconds", !this.duration_options.show_seconds);
```

Snake case for variables is de-facto in Frappe, use it unless absolutely necessary.